src/vaishnaviVJun20/Math.java

```diff
@@ -0,0 +1,40 @@
+package vaishnaviVJun20;
+
+public class Math {
+    double sum(double variable1, double variable2, double variable3) {
```

Comment:
It looks better if we use number1 instead of variable1

Reply:
Done